bench_method_test.go

```diff
@@ -54,24 +55,18 @@ func benchmarkMethodForTest(t *testing.T, methodString string, p transport.Proto
 }
 
 func TestBenchmarkMethodWarmTransport(t *testing.T) {
-	s := newServer(t)
+	s := newServer(t, nil)
 	defer s.shutdown()
 	s.register(fooMethod, methods.echo())
 
 	tests := []struct {
-		hostPort       string
-		callerOverride string
-		method         string
-		wantErr        string
+		hostPort string
+		method   string
+		wantErr  string
 	}{
 		{
 			hostPort: s.hostPort(),
 		},
-		{
```

Comment:
This is a far flung effect of refactoring tracing. Tracing is no longer a transport level concern, since we no longer use trace sampling on the channel level. Consequently it becomes a request scoped concern and the logic to detect whether the caller was overridden had to be factored upward so I could validate benchmark options.